src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -188,14 +188,14 @@ public override void _Draw()
         if (connections.Count == 0)
         {
             // Clear existing connection lines
-            lineContainer.FreeChildren();
+            lineContainer.QueueFreeChildren();
```

Comment:
Does this need to detach the nodes here? That I think costs more performance than passing in false here.

Reply:
I forgot to add `false` :sob:

Reply:
Seems like a simple enough change so I'll merge this without playtesting again as if anything the performance should be even better with this.